docs/tool/js/views/filter-view.js

```diff
@@ -191,7 +219,93 @@ var filterView = {
         }
 
         this.isTouched = function(){
-            return slider.noUiSlider.get()[0] === c.min && slider.noUiSlider.get()[1] === c.max;
+            return slider.noUiSlider.get()[0] !== c.min && slider.noUiSlider.get()[1] !== c.max;
+        }
+
+    },
+    dateFilterControl: function(component){
+        filterView.filterControl.call(this, component);
```

Comment:
Only comment for now on a quick skim - you should switch out the appending of title text that is done step by step below for a call to the `filterView.setupFilter(c);` function that is used in the other controls. I refactored it this way so that we wouldn't duplicate the setup of title with click functions to open/close accordion etc.. It requires a slight change to how you actually append the slider itself, since setupFilter returns the "content" div